Library/Formula/ice.rb

```diff
@@ -13,26 +10,19 @@ class Ice < Formula
     sha1 "7bdb347052b8df900c2c2e6797726a79d9c0242d" => :mountain_lion
   end
 
-  option 'doc', 'Install documentation'
-  option 'demo', 'Build demos'
+  option "without-java", "Compile without Java support."
+
+  depends_on :java => "1.7" if build.with? "java"
 
```

Comment:
@DomT4 Isn't that a dependency on a specific version, rather than all versions greater than some version?

I've used the above here:
https://github.com/Homebrew/homebrew-science/pull/2326/files#diff-0cdaeb4fa14b9af05dd33275e4359ff1R24
and it worked fine there.  Not sure what the difference is here.  It's in
https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/minimum_macos_requirement.rb so not sure why it's not working here.